lesson_2/calculator.py

```diff
@@ -24,12 +52,9 @@
             output = 'Error: Cannot divide by zero!'
         else:
             output = num1 / num2
```

Comment:
nit: Same suggestion here about making this a function, it will make this code cleaner/more readable.

Reply:
not clear what you are referencing should be in a function? the output = num1 (operator) num2 or just line ? @Doxify